phpstan.dist.neon

```diff
@@ -20,10 +20,11 @@ parameters:
         # ignore PHPUnit data providers return type as they are not checked against the test methods anyway
         - '#^Method PhpCsFixer\\Tests\\.+::provide.+Cases\(\) return type has no value type specified in iterable type (array|iterable)\.$#'
 
+        # Architectural assertions (e.g. ensuring inheritance, composition)
```

Comment:
this looks like very narrow scope for quite wide regex

Reply:
This comment defines section, architectural rules should be here, while code-related above it. It's not needed, I just thought it could be better to split it. I can remove it 🙂.

Reply:
I removed the comment.